server.go

```diff
@@ -3175,6 +3192,24 @@ func (s *server) cancelConnReqs(pubStr string, skip *uint64) {
 	delete(s.persistentConnReqs, pubStr)
 }
 
+// handleCustomMessage dispatches an incoming custom peers message to
+// subscribers.
+func (s *server) handleCustomMessage(peer [33]byte, msg *lnwire.Custom) error {
+	srvrLog.Debugf("Custom message received: peer=%x, type=%d",
+		peer, msg.Type)
+
+	return s.customMessageServer.SendUpdate(&CustomMessage{
```

Comment:
Pretty sure this will crash if the `customMessageServer` isn't set

Reply:
How do you mean 'why the nil-check', there is none? This callback is never supposed to be nil, so I don't think a nil check is needed. Another config delegate `PrunePersistentPeerConnection` isn't nil-checked either for example.

Reply:
Here https://github.com/lightningnetwork/lnd/pull/5346/files#diff-366e46a40f6f60b4f7614eb0976bb51820364bf5ca6ccc4787eb49d7bdbef3e6R1502

Reply:
Ah got it. Sorry, was looking at `Brontide`. Fixed.